spec/system/school_onboarding_spec.rb

```diff
@@ -44,6 +45,9 @@
       expect(page).to have_select('Weather Underground Data Feed Area', selected: 'BANES weather')
       expect(page).to have_select('Solar PV from The University of Sheffield Data Feed Area', selected: 'BANES solar')
 
+      # Why does this fail?!
+      # expect(page).to have_select('Dark Sky Data Feed Area', selected: 'BANES dark sky weather')
```

Comment:
This is doing my nut! Any ideas @jamesalmond - I can't see the wood for the trees!